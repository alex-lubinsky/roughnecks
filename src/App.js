import React from 'react';
import CheckmarkDashboard from './components/CheckmarkDashboard';
import './styles/App.css';


class App extends React.Component {

  render() {
    return (
    <div>
      <CheckmarkDashboard fallen={false} />
    </div>
    )
  }
}

export default App;