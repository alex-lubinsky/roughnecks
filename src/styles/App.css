@import "ValidationMessage.css";

.icon-ok,
.icon-legal,
.icon-ok-circle {
  color: green;
}

.icon-remove,
.icon-remove-sign {
  color: red;
}

.div-margin-sm {
  margin: 20px;
}

.checkmark-dashboard tr:nth-child(4n+3) {
  background: rgb(240, 240, 240);
}

.character-row {
  border-bottom: 1px solid rgb(175, 174, 174);
  border-top: 1px solid rgb(175, 174, 174);
}

.character-row td {
  padding: 0px 10px;
}

.spacer td {
  padding: 5px;
}

table th, table td {
  padding: 5px;
  text-align: center;
}