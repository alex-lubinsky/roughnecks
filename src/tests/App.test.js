import App from "../App";
import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("App Component", () => {
  test("renders correctly", () => {
    const wrapper = shallow(<App />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
