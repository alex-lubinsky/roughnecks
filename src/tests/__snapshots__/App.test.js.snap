// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders correctly 1`] = `
<div>
  <Connect(CheckmarkDashboard)
    fallen={false}
  />
</div>
`;
